selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/CheckRestoringOfSplitEditorTest.java

```diff
@@ -23,34 +21,34 @@
 import java.util.List;
 import org.eclipse.che.commons.lang.NameGenerator;
 import org.eclipse.che.commons.lang.Pair;
+import org.eclipse.che.selenium.core.SeleniumWebDriver;
 import org.eclipse.che.selenium.core.client.TestProjectServiceClient;
+import org.eclipse.che.selenium.core.constant.TestTimeoutsConstants;
 import org.eclipse.che.selenium.core.project.ProjectTemplates;
 import org.eclipse.che.selenium.core.workspace.TestWorkspace;
 import org.eclipse.che.selenium.pageobject.CodenvyEditor;
 import org.eclipse.che.selenium.pageobject.Ide;
 import org.eclipse.che.selenium.pageobject.Loader;
+import org.eclipse.che.selenium.pageobject.NotificationsPopupPanel;
 import org.eclipse.che.selenium.pageobject.PopupDialogsBrowser;
 import org.eclipse.che.selenium.pageobject.ProjectExplorer;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 /** @author Musienko Maxim */
-public class CheckRestoringOfSplitEditor {
-  private String nameJavaClass = "AppController.java";
-  private String nameReadmeFile = "README.md";
-  private String namePomFile = "pom.xml";
-  private String nameOfTabPomFile = "qa-spring-sample";
+public class CheckRestoringOfSplitEditorTest {
+  private String javaClassName = "AppController.java";
+  private String readmeFileName = "README.md";
+  private String pomFileTab = "qa-spring-sample";
+  private String javaClassTab = "AppController";
   private final String PROJECT_NAME =
-      NameGenerator.generate(CheckRestoringOfSplitEditor.class.getSimpleName(), 4);
+      NameGenerator.generate(CheckRestoringOfSplitEditorTest.class.getSimpleName(), 4);
```

Comment:
How many times I should say to use 
`NameGenerator.generate("project", 4);`
for project names?